css/style.css

```diff
@@ -1,3 +1,62 @@
-h1 {
-  color: salmon;
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  background-color: #f2f2f2;
+}
+.section-a {display: flex;
+justify-content: center;
+
+}
+
+h1{
+color: #4c4c4c;
+-webkit-margin-after: 0px;
+-webkit-margin-before: 0px;
+}
+
+p{ color: #666666;
+  -webkit-margin-after: 30px;
+  -webkit-margin-before: 0px;
+}
+
+.section-b {display: flex;
+justify-content: center;
+}
+
+img {
```

Comment:
Be careful using selectors that are potentially too broad. When you specify the `img` selector every `<img>` tag across the whole site will be affected. In a larger site that might mean every use of `img` from article photos to social media icons will get the same size and style properties.

It's usually best to err on the side of being more specific. In this case `.container img` would only style the images inside of `.container` elements.